README.md

```diff
@@ -3,4 +3,9 @@
 Check the Github Actions workflow for how to build the code.
 Check https://github.com/solvcon/mmnote for the theoretical and numerical notes.
 
+## Binaries
```

Comment:
Please be specific that this is for Windows binary.  "Windows portable binaries" reads clearer.

Reply:
personally, I prefer "Download", which is more common.

Reply:
Download sounds good